### ISSUE TITLE
"Similar and Related Adventures" missing some expected adventures

### ISSUE THREAD

**Issue by Wigginns** [AdventureLookup/AdventureLookup#377]
#365 added functionality to see similar adventures. Largely this is successful and works well. I noticed some odd behavior with a few adventures specifically when browsing for an adventure. The skyhorn lighthouse series seems to have a small hangup. No common adventures are found though they both have "skyhorn lighthouse" in the title. 

https://adventurelookup.com/adventures/the-secrets-of-skyhorn-lighthouse
https://adventurelookup.com/adventures/the-corruption-of-skyhorn-lighthouse

![image](https://user-images.githubusercontent.com/3180645/86974761-1bf93b00-c134-11ea-9d0e-ca842ea5f9fb.png)

The search terms for these seem a little odd. It appears to be truncating the common letters are the end of unique terms.

**Comment by Wigginns** [comment]
I am just now noticing that many adventures seem to have a similar hangup. I'm guessing it may be an elasticsearch reindexing issue though I haven't had a change to begin to dig 🙃

**Comment by cmfcmf** [comment]
> I am just now noticing that many adventures seem to have a similar hangup. I'm guessing it may be an elasticsearch reindexing issue though I haven't had a change to begin to dig

I don't think it necessarily is that. It could also be the approach we use to find similar adventures. If you find any other cases of similar adventures that are not marked as similar, please also post the urls of the adventures. It helps a lot to have examples of where it doesn't work.
